### ISSUE TITLE
question: Is there a way to set the prefix?

### ISSUE THREAD

**Issue by jetjinser** [dmitmel/cmp-digraphs#1]
as the title

**Comment by rstacruz** [comment]
Hey there, new user here - anyone know what the prefix is? I can't figure out how to trigger it.

**Comment by dmitmel** [comment]
What do you mean by the prefix?

**Comment by jetjinser** [comment]
just like [cmp-latex-symbols](https://github.com/kdheepak/cmp-latex-symbols), just when `\` typed provide digraphs

**Comment by aemonge** [comment]
Yeah.... since right now, it just removes all the other plugins, and it's a bit wrong. Ideally, I would love to be combined with emojis, and just trigger them jointly with a `:` prefix.

https://github.com/hrsh7th/cmp-emoji
